lua/nvim-treesitter/locals.lua

```diff
@@ -126,7 +126,7 @@ function M.get_capture_matches(bufnr, capture_string, query_kind)
 
     local matches = {}
     for _, match in pairs(M.get_locals(bufnr, query_kind)) do
-      local insert = utils.get_at_path(match, capture_string..'.node')
+      local insert = utils.get_at_path(match, capture_string)
```

Comment:
Why is this needed ?

Reply:
I need the whole capture without node. It also contains my starts. I had this also changed for query things like `strip!`
A @function.outer match would look like that
```lua 
{
    node = <userdata>,
    start = {
              node = <userdata>
     }
}